### PR TITLE
helpers: editBotVoiceState

### PR DESCRIPTION
**Pull request by lts20050703**
Closes: #2409 
Upstream: https://github.com/discord/discord-api-docs/commit/631b4d4a14ace4c427d432460475d3e5970feb46

### CODE REVIEW

File: helpers/misc/editBotVoiceState.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Bot } from "../../bot.ts";
+
+export function editBotVoiceState(
```
Comment (Skillz4Killz) [inline helpers/misc/editBotVoiceState.ts:3]: Need comment above this /**

File: helpers/misc/editBotVoiceState.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  bot: Bot,
+  guildId: bigint,
+  options: { channelId?: bigint; suppress?: boolean; requestToSpeakTimestamp?: number },
+) {
+  return bot.rest.runMethod<void>(bot.rest, "PATCH", bot.constants.routes.EDIT_BOT_VOICE_STATE(guildId), {
+    channel_id: options.channelId?.toString(),
+    suppress: options.suppress,
+    request_to_speak_timestamp: options.requestToSpeakTimestamp
+      ? new Date(options.requestToSpeakTimestamp).toUTCString()
```
Comment (Skillz4Killz) [inline helpers/misc/editBotVoiceState.ts:12]: UTC? i thought discord required ISO strings?

Comment (lts20050703) [reply helpers/misc/editBotVoiceState.ts:12]: whoops my bad

**Review by Skillz4Killz — COMMENTED** [review]
RIP i forgot to click submit on this review

File: helpers/misc/editBotVoiceState.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { Bot } from "../../bot.ts";
+
+/** Updates the current user's voice state. `channelId` must currently point to a stage channel. Bot must already have joined `channelId`. You must have the `MUTE_MEMBERS` permission to unsuppress yourself. You can always suppress yourself. You must have the `REQUEST_TO_SPEAK` permission to request to speak. You can always clear your own request to speak. You are able to set `requestToSpeakTimestamp` to any present or future time.
```
Comment (Skillz4Killz) [inline helpers/misc/editBotVoiceState.ts:3]: Needs jsdocs like other helpers

Comment (lts20050703) [reply helpers/misc/editBotVoiceState.ts:3]: @Skillz4Killz Sorry what? Isn't that JSDocs comments?

**Comment by vxern** [comment]
`editBotVoiceState.ts` is already found in `/helpers/guilds/voice`. There is also no need for moving `editBotMember.ts` from `/helpers/members` to `/helpers/misc`.

**Pull request closed without merging**